lib/core-server/src/build-dev.ts

```diff
@@ -79,19 +79,20 @@ export async function buildDevStandalone(options: CLIOptions & LoadOptions & Bui
     fullOptions
   );
 
-  // @ts-ignore
-  const previewTotalTime = previewResult?.totalTime;
-  // @ts-ignore
-  const managerTotalTime = managerResult?.totalTime;
+  const previewTotalTime = previewResult && previewResult.totalTime;
```

Comment:
why?

Reply:
Why did I get rid of the unnecessary `@ts-ignore`?

Reply:
why is this

```
previewResult && previewResult.totalTime
```

better than this

```
previewResult?.totalTime
```

Reply:
Not sure. I suspect our version of ts doesn't support optional chaining?

Reply:
It looks like the type of `previewResult` is a bit strange (see the `void | XXX`):

```ts
const previewResult: void | {stats: unknown, totalTime: [number, number], bail: (e?: Error) => Promise<void>}
```

And so, in `previewResult?.totalTime` there is a TS error because there is no `totalTime` on type `void`.

On the other hand with `previewResult && previewResult.totalTime` the first `previewResult &&` allows TS to infer the type of `previewResult` more precisely (i.e. without the `void | `) for the rest of the expression.

The proper solution for this would be to define a proper return type to https://github.com/storybookjs/storybook/blob/7ef4e0e7c0c0befd1a9acf7546d9f36d5b34dc1b/lib/core-server/src/dev-server.ts#L18 and handle edge cases in it.